src/components/ShowCard.tsx

```diff
@@ -41,9 +39,11 @@ export default function ShowCard(props: MovieCardProps): JSX.Element {
 						{/* TODO: Include number of stars with styling, response returns rating out of 10  */}
 						<p>{props.details.vote_average} stars</p>
 						<span>{props.details.vote_count} ratings</span>
-						<div>
-							{ratingHandler(props.details)}
-						</div>
+						{ratingHandler && (
```

Comment:
I could be wrong but I think we need to check `ratingHandler(props.details)` here. 

Since that would end up running the function twice, it might be better to just set a state in the rating handler and render according to that.

This could be added to a future ticket or something, definitely not a bit deal.